benchmarks/clhqueuebench.c

```diff
@@ -19,7 +19,15 @@ CLHLockStruct *lhead, *ltail;
 Node guard CACHE_ALIGN = {GUARD_VALUE, NULL};
 
 volatile Node *Head CACHE_ALIGN = &guard;
+#ifdef DEBUG
+volatile uint64_t deq_state = 0;
+#endif
```

Comment:
You can merge the `#ifdef DEBUG` sections into one.

Reply:
The deq_state is declared just after Head (which is cache-aligned). Thus, touching both deq_state and Head from dequeuers does not causes any extra cache-misses.

Similarly, enq_state is just after Tail in order to not cause additional cache-misses.